core/src/com/mygdx/config/GameConfigs.java

```diff
@@ -0,0 +1,51 @@
+package com.mygdx.config;
+
+import com.badlogic.gdx.utils.Array;
+import lombok.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GameConfigs {
+
+    @NonNull
+    private final Map<Class<? extends EntityConfig>, Map<String, EntityConfig>> entityConfigListMap;
```

Comment:
To tylko taki komentarz, ciekawe, nie wiedziałem, że da się za klucz dać klasę :D. Nazwę bym zmienił na entityConfigMap

Reply:
no nazwa była taka, że to przez chwilę była mapa do list, ale uznałem, że jednak wolę mapę